HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -746,22 +748,23 @@ namespace WPEFramework
 		   Register(HDMICECSINK_METHOD_SET_LATENCY_INFO, &HdmiCecSink::setLatencyInfoWrapper, this);
            logicalAddressDeviceType = "None";
            logicalAddress = 0xFF;
+           // load persistence setting
+           loadSettings();
+
+           int err;
+           dsHdmiInGetNumberOfInputsParam_t hdmiInput;
+           InitializeIARM();
```

Comment:
Did you add any new IARM registration?  Or this initialize was not called before?